anghammarad/src/main/scala/com/gu/anghammarad/messages/HangoutsService.scala

```diff
@@ -8,15 +8,15 @@ import sttp.client3._
 
 object HangoutsService {
 
-  def postMessage(webhook: String, message: String): Response[Either[String, String]] = {
+  def postMessage(webhook: String, message: String): Identity[Response[Either[String, String]]] = {
```

Comment:
Could you explain the `Identity` addition here? Having `Identity[Response[Either...` feels like a lot of nesting.

Reply:
ah - thats a mistake - I think it must've crept in when I was rebasing - I'll get rid of those now.